vnext/ReactUWP/Views/FrameworkElementViewManager.cpp

```diff
@@ -270,6 +274,31 @@ void FrameworkElementViewManager::UpdateProperties(ShadowNodeBase* nodeToUpdate,
             winrt::AutomationProperties::SetAccessibilityView(element, winrt::Peers::AccessibilityView::Raw);
         }
       }
+      else if (propertyName == "accessibilityLiveRegion")
+      {
+        if (propertyValue.isString())
+        {
+          auto value = propertyValue.asString();
```

Comment:
You'll want to use getString here to avoid the type checking overhead since you already know it's a string.

Reply:
Fixed